components/fieldlabel/skin.css

```diff
@@ -10,7 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-.spectrum-FieldLabel,
+/* .spectrum-FieldLabel,
```

Comment:
@pfulton  should I use here core tokens colors? Or is the inheritance here ok?

Reply:
We should use the core tokens here. Also, we need to delete this `skin.css` file, and put the color stuff into the `index.css`